lib/verify.js

```diff
@@ -15,7 +15,7 @@ const canBeDisabled = (validator) => (value) => value === false || validator(val
 
 const VALIDATORS = {
   assets: isArrayOf(
-    (asset) => isStringOrStringArray(asset) || (isPlainObject(asset) && isStringOrStringArray(asset.path))
+    (asset) => isStringOrStringArray(asset) || (isPlainObject(asset) && isNonEmptyString(asset.url)) || (isPlainObject(asset) && isStringOrStringArray(asset.path))
```

Comment:
What about adding some brackets for readability and getting rid of the double `isPlainObject` call?

```suggestion
    (asset) => isStringOrStringArray(asset) || (isPlainObject(asset) && (isNonEmptyString(asset.url) || isStringOrStringArray(asset.path)))
```

Reply:
Done